### DISCUSSION TITLE
Failing to publish to npm when package's README has ")AND1<2" in it

### DISCUSSION THREAD

**Discussion by gravitypersists** [npm/feedback#817] [CLI]
I had tried publishing a package, via cli, with the following string in its README: 

`(3 > 2) AND (3 < 10)` 

It failed, reporting a 403. Removing that string from the README solved the issue.

I diagnosed the issue a bit further and reduced it down to anything matching `\)AND\d\<\d`will produce this error. So having the string `)AND1<2` will also fail to publish. 

(I wouldn't be surprised if my organization just has a funny regex security rule configured into npm, if that exists... but I thought I'd report it anyways because I imagine it's a pervasive little bug that will rarely be crossed)

**Comment by ljharb** [comment]
readmes are markdown, which accept HTML, so you have to escape `<` and `>` anyways. Have you tried that?

**Reply by gravitypersists** [reply]
Yes, this fails when you put it within backticks as well. The code I saw, which ran into this, was in backticks like so:

```
  To achieve a "between" operation, combine with AND, `(3 > 2) AND (3 < 10)`
```

**Reply by ljharb** [reply]
No, not backticks, *escaped*. Like `&lt;` and `&rt;`.

**Reply by arcanis** [reply]
> readmes are markdown, which accept HTML, so you have to escape < and > anyways

It's not mandatory; Markdown [autoescapes](https://daringfireball.net/projects/markdown/syntax#autoescape):

> Markdown allows you to use these characters naturally, taking care of all the necessary escaping for you. If you use an ampersand as part of an HTML entity, it remains unchanged; otherwise it will be translated into `&amp;`.

> Similarly, because Markdown supports [inline HTML](https://daringfireball.net/projects/markdown/syntax#html), if you use angle brackets as delimiters for HTML tags, Markdown will treat them as such. But if you write `4 < 5`, Markdown will translate it to`4 &lt; 5`

As long as `&`, `<`, and `>` are surrounded by spaces, it shouldn't be a problem.

**Reply by ljharb** [reply]
ah, thanks for clarifying.

**Reply by gravitypersists** [reply]
I would assume any markdown engine does not try to parse characters within backticks as tags...

But fwiw, no `) AND (3 &lt; 10)` does not experience this error. But it does look like this when you look at it:

![image](https://user-images.githubusercontent.com/231863/205358385-1136f1a7-39c2-4a86-a41c-a917ce37ee42.png)

**Reply by ljharb** [reply]
Sounds like it's indeed a bug, then :-)
